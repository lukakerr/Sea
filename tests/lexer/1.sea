num n1 = 5
num n2 = 3
num n3 = 5 % 3
num n4 = n3 + n1
num n5 = n2 - 1
num n6 = n2 * 1
num n7 = 3 / 3
num n8 = 10 % %
n1 += 1
n2 -= 2
n3 *= 3
n4 /= 4
n5 %= 5