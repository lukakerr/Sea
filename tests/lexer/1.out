[DataType N,Identifier "n1",Operator Eq,Num 5,DataType N,Identifier "n2",Operator Eq,Num 3,DataType N,Identifier "n3",Operator Eq,Num 5,Operator Modulus,Num 3,DataType N,Identifier "n4",Operator Eq,Identifier "n3",Operator Plus,Identifier "n1",DataType N,Identifier "n5",Operator Eq,Identifier "n2",Operator Minus,Num 1,DataType N,Identifier "n6",Operator Eq,Identifier "n2",Operator Times,Num 1,DataType N,Identifier "n7",Operator Eq,Num 3,Operator Divide,Num 3,DataType N,Identifier "n8",Operator Eq,Num 10,Operator Modulus,Operator Modulus,Identifier "n1",Operator PlusEq,Num 1,Identifier "n2",Operator MinusEq,Num 2,Identifier "n3",Operator TimesEq,Num 3,Identifier "n4",Operator DivideEq,Num 4,Identifier "n5",Operator ModulusEq,Num 5]
